assets/js/modules/ads/pax/services.test.js

```diff
@@ -73,7 +73,7 @@ describe( 'PAX partner services', () => {
 							},
 						},
 					};
-					services = createPaxServices( registry, {
+					services = createPaxServices( registry, null, {
```

Comment:
Having to supply `null` as an argument like this isn't a great API. Let's change the second argument to be an object instead, since the arguments are all optional. We shouldn't have optional positional arguments 🙂

Reply:
Yeah that's a good point